### PR TITLE
Bug 1263173 - Use bootstrap-style close button for graph popup

### PR DESCRIPTION
**Pull request by nihal111**
The graph popup close button as you requested @wlach

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1498)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by nihal111** [comment]
This only makes sense with my prev PR #1497 which uses the changed CSS code.

**Comment by wlach** [comment]
I think you forgot to include the controller/html changes in your commit?

File: ui/css/perf.css [hunk 1]
```diff
@@ -106,10 +106,13 @@ to { opacity: 0; }
     background-image: url("../img/tip-locked.png");
 }
 
-#close-popup {
-  float:right;
-  display:inline-block;
-  cursor:pointer;
+.graphchooser-close {
+  color: #fff;
+  outline: none;
```
Comment (wlach) [inline ui/css/perf.css]: Could you add a `margin: -14px -4px;` here. I think it looks a little better up and to the right.

**Comment by wlach** [comment]
One minor change request. This is looking good!

**Comment by nihal111** [comment]
Done. @wlach

**Comment by wlach** [comment]
Thanks!

**Comment by edmorley** [comment]
The commit message for this commit was the same as the one for #1497? ("Outside click clears popup")

**Comment by wlach** [comment]
Yeah sorry the commit message was wrong, should have noticed that. @nihal111 -- in the future, commit messages should always have the format "Bug 1234 - Thing that was changed"

**Pull request merged**